service/matching/liveness_test.go

```diff
@@ -70,14 +70,14 @@ func (s *livenessSuite) TearDownTest() {
 
 func (s *livenessSuite) TestIsAlive_No() {
 	liveness := newLiveness(s.timeSource, s.ttl, func() { atomic.CompareAndSwapInt32(&s.shutdownFlag, 0, 1) })
-	s.timeSource.Update(time.Now().Add(s.ttl * 2))
+	s.timeSource.Advance(s.ttl * 2)
 	alive := liveness.isAlive()
 	s.False(alive)
 }
 
 func (s *livenessSuite) TestIsAlive_Yes() {
 	liveness := newLiveness(s.timeSource, s.ttl, func() { atomic.CompareAndSwapInt32(&s.shutdownFlag, 0, 1) })
-	s.timeSource.Update(time.Now().Add(s.ttl / 2))
```

Comment:
these kinds of changes are going to make things a lot easier to test

Reply:
Do you mean time.Update is easier? Unfortunately, clockwork does not provide a way to set time. Though, normally, you don't need to set time, you need to advance to a certain point that is required for some logic to happen.

Reply:
no, this is a " :+1: good API" - advancing by more obvious deltas is a lot more readable than mathing out multiple "set time to X" calls